packages/testing/tests/analytics-sales-report.test.js

```diff
@@ -15,39 +15,49 @@ fixture`Analytics Sales Report`;
 
 [false, true].forEach((embedded) => {
   screenModes.forEach((screenMode) => {
-    test(`Analytics Sales Report (${project}, embed=${embedded}, ${screenMode[0]})`, async (t) => {
-      const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
-      await toggleCommonConfiguration(t, BASE_URL, embedded, () => { },
-        screenMode, timeoutSecond, false, requestLogger);
-      await forceResizeRecalculation(t, screenMode);
-      await t.expect(Selector('body.dx-device-generic').count).eql(1);
-      await takeScreenshot(`analytics-sales-report-month${getPostfix(embedded, screenMode)}`, 'body');
-
-      const isPeriodSelectorBoxVisible = screenModes[0] === screenMode;
-      if (isPeriodSelectorBoxVisible) {
-        await t.click(Selector('.sales-filter .dx-dropdownbutton'));
-
-        await t.click(Selector('.dx-dropdownbutton-popup-wrapper .dx-list .dx-list-item').nth(0));
-        await t.wait(timeoutSecond);
-        await takeScreenshot(`analytics-sales-report-day${getPostfix(embedded, screenMode)}`, 'body');
+    themeModes.forEach((themeMode) => {
+      const postfix = getPostfix(embedded, screenMode, themeMode);
+
+      if (embedded && themeMode === 'dark') {
```

Comment:
Why do you skip dark theme here?

Reply:
We should always ignore the dark theme in the embedded mode because we change themes by clicking a button that does not appear in the embedded mode.

We should enhance the method that changes themes (teach it to switch themes via code somehow) or skip dark themes when embedded as we currently do.

I think the current skipping is conceptually okay: embedded tests in light mode check that the layout is not broken when embedded, while it looks like it is enough to run tests for dark mode in non-embedded mode to check color issues